test/src/search/code_controller_test.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import '../common/create_app.dart';
+
+void main() {
+  testWidgets('CTRL + F or META + F enables search', (wt) async {
```

Comment:
Wrap in group 'CodeSearchController'.
```suggestion
  testWidgets('CTRL + F shows search, Escape hides', (wt) async {
```
Merge with the test below.